pages/education.py

```diff
@@ -201,73 +300,53 @@ def total_edu_figure(df):
-            y=df.real_expenditure_centralized,
-            marker_color="rgb(17, 141, 255)",
-        ),
-    )
-    fig.add_trace(
-        go.Bar(
-            name="Regional",
-            x=df.year,
-            y=df.real_expenditure_decentralized,
-            marker_color="rgb(160, 209, 255)",
-        ),
-    )
-
-    fig.update_xaxes(tickformat="d")
-    fig.update_yaxes(fixedrange=True)
-    fig.update_layout(
-        barmode="stack",
-        hovermode="x",
-        title="How has total expenditure changed over time?",
-        plot_bgcolor="white",
-        legend=dict(orientation="h", yanchor="bottom", y=1),
-        annotations=[
-            dict(
-                xref="paper",
-                yref="paper",
-                x=-0.14,
-                y=-0.2,
-                text="Source: BOOST & CPI: World Bank",
-                showarrow=False,
-                font=dict(size=12),
+    try:
+        spending = pd.DataFrame(data["edu_public_expenditure"])
+        spending = filter_country_sort_year(spending, country)
+
+        start_year = spending.year.min()
+        end_year = spending.year.max()
+        start_value = spending[spending.year == start_year].real_expenditure.values[0]
+        end_value = spending[spending.year == end_year].real_expenditure.values[0]
+        start_value_central = spending[
+            spending.year == start_year
+        ].central_expenditure.values[0]
+        end_value_central = spending[
+            spending.year == end_year
+        ].central_expenditure.values[0]
+        start_value_decentralized = spending[
+            spending.year == start_year
+        ].decentralized_expenditure.values[0]
+        end_value_decentralized = spending[
+            spending.year == end_year
+        ].decentralized_expenditure.values[0]
+
+        spending_growth_rate = ((end_value - start_value) / start_value) * 100
+        spending_growth_rate_central = (
+            (end_value_central - start_value_central) / start_value_central
+        ) * 100
+        spending_growth_rate_decentralized = (
+            (end_value_decentralized - start_value_decentralized)
+            / start_value_decentralized
+        ) * 100
+
+        text = f"""Between {start_year} and {end_year}, the total real public spending on education in {country} has increased from ${millify(start_value)} to ${millify(end_value)} reflecting the growth rate of {spending_growth_rate:.2f}% after adjusting for the inflation rate. \n
+            During the same time period, the central government's spending has {get_percentage_change_text(spending_growth_rate_central)} """
+
+        if not np.isnan(spending_growth_rate_decentralized):
+            decentralized_spending_text = f"while the local gov't's spending has  {get_percentage_change_text(spending_growth_rate_decentralized)}."
+        else:
+            decentralized_spending_text = (
+                ". The local gov't's data is not available for this period."
```

Comment:
```suggestion
                ". The local government's data is not available for this period."
```
since gov't's looks a bit odd